### PR TITLE
Fix/transition emails

### PR DESCRIPTION
**Pull request by allenjprice**
removes all placeholder copy from transition-related automated emails

### CODE REVIEW

**Comment by jalehman** [comment]
I'm going to hold off on merging this. Will merge if the Tipe migration won't be finished before EOD.

File: src/reactor/handlers/license_transition.clj [hunk 1]
```diff
@@ -210,27 +251,40 @@
...
+  "The Tipe document id for the intra-xfer-created email template"
+  "5b2183eba83135001307db90")
+
+
+(defn prepare-intra-xfer-created-email
+  [document member unit new-license current-license]
+  (tb/transform-when-key-exists document
+    {:subject (fn [subject] (stache/render subject {:name (account/first-name member)}))
+     :body (fn [body] (-> (stache/render body {:name (account/first-name member)
+                                              :new-unit (make-friendly-unit-name (member-license/unit new-license))
```
Comment (jalehman) [inline src/reactor/handlers/license_transition.clj]: indentation

File: src/reactor/handlers/license_transition.clj [hunk 2]
```diff
@@ -332,26 +397,52 @@
...
-      (mm/p
-       (format "Your new license will take effect on %s. This license is effective for a %s month term at a rate of %s/month. We will continue to roll your license over on a month-to-month basis until we've received your notice of intent to move out." (date/short (member-license/starts new-license)) (member-license/term new-license) (member-license/rate new-license)))
-      (mm/p "If you have any questions, please don't hesitate to ask your community representative.")
-      (mm/sig))
-     {:uuid (event/uuid event)})))
+     (mail/subject (:subject content))
+     (mm/msg (:body content))
+     {:uuid (event/uuid event)})
+
+    #_(mailer/send
```
Comment (jalehman) [inline src/reactor/handlers/license_transition.clj]: remove

File: src/reactor/handlers/license_transition.clj [hunk 3]
```diff
@@ -260,14 +301,14 @@
...
-                                              :old-unit (make-friendly-unit-name unit)
-                                              :move-out-date (date/short (member-license/ends current-license))
-                                              :move-in-date (date/short (member-license/starts new-license))
-                                              :term (str (member-license/term new-license))
-                                              :rate (str (member-license/rate new-license))})
-                         (md/md-to-html-string)))}))
+     :body    (fn [body] (-> (stache/render body {:name          (account/first-name member)
+                                                 :new-unit      (make-friendly-unit-name (member-license/unit new-license))
+                                                 :old-unit      (make-friendly-unit-name unit)
+                                                 :move-out-date (date/short (member-license/ends current-license))
```
Comment (jalehman) [inline src/reactor/handlers/license_transition.clj]: time zone correction

File: src/reactor/handlers/license_transition.clj [hunk 4]
```diff
@@ -335,14 +376,14 @@
...
```
Comment (jalehman) [inline src/reactor/handlers/license_transition.clj]: time zone correction

File: src/reactor/handlers/license_transition.clj [hunk 5]
```diff
@@ -76,15 +78,42 @@
...
+(defn prepare-move-out-before-30-days-email
+  [document member admin transition]
+  (let [license (license-transition/current-license transition)
+        tz      (member-license/time-zone license)]
+    (tb/transform-when-key-exists document
+      {:subject (fn [subject] (stache/render subject {:name (account/first-name member)}))
+       :body    (fn [body] (-> (stache/render
+                               body {:name                  (account/first-name member)
+                                     :community-team-member (account/first-name admin)
+                                     :move-out-date         (date/short (date/tz-corrected (license-transition/date transition) tz))
```
Comment (jalehman) [inline src/reactor/handlers/license_transition.clj]: might be helpful to define a function in this namespace such as `format-date` that takes a date and a timezone and does the `date/short` and `date/tz-corrected` stuff.

**Comment by allenjprice** [comment]
should I be using `tz-corrected` or `tz-uncorrected`? the second one is giving me the correct dates, but that feels incorrect

**Pull request merged**